scripts/quests/bastok/The_Elvaan_Goldsmith.lua

```diff
@@ -40,7 +42,8 @@ quest.sections =
                 (
                     status == xi.questStatus.QUEST_COMPLETED and
                     player:getFameLevel(xi.fameArea.BASTOK) == 1 and
-                    not quest:getMustZone(player)
+                    not quest:getMustZone(player) or
```

Comment:
This logic looks a bit loose by tacking that or on in the end of 3 and conditions.  Even if not necessary, it should probably be clarified with parens